ai_presenter/text_ai/chat/messages.py

```diff
@@ -0,0 +1,34 @@
+# import logging
+# import openai
+import tiktoken
+import json
+
+global MAX_TOKEN_LIMIT
+global SOFT_TOKEN_LIMIT
+MAX_TOKEN_LIMIT = 4096
+SOFT_TOKEN_LIMIT = 2500
+
+
+class Messages:
+    def __init__(self, gpt_initializer):
+        self.gpt_setup = gpt_initializer
+        self.scenes = []
+        self.max_token_limit = MAX_TOKEN_LIMIT
+        self.soft_token_limit = SOFT_TOKEN_LIMIT
+
+    def update_scenes(self, scene):
```

Comment:
Instead of scene, take in `request`, and `response` so that the class can save it together to make it easier to construct later.